presto-docs/src/main/sphinx/functions/qdigest.rst

```diff
@@ -22,10 +22,10 @@ Functions
 
 .. function:: value_at_quantile(qdigest(T), quantile) -> T
 
-    Returns the approximate percentile values from the quantile digest given
+    Returns the approximate percentile value from the quantile digest given
     the number ``quantile`` between 0 and 1.
 
-.. function:: values_at_quantiles(qdigest(T), quantiles) -> T
+.. function:: values_at_quantiles(qdigest(T), quantiles) -> array<T>
```

Comment:
The canonical form for array and other structural types uses parenthesis instead of `<>`: `array(T)`